### DISCUSSION TITLE
[Support]: Have the GHCR container names changed?

### DISCUSSION THREAD

**Discussion by victorhooi** [blakeblackshear/frigate#13450] [General Support]
### Describe the problem you are having

Previously, there were Docker image tags on GHCR of the form `dev-99e0357-standard-arm64` etc.

However, if I look on GHCR now (https://github.com/blakeblackshear/frigate/pkgs/container/frigate), there's no more `dev` images and they all just seem to be named after hashes?

![CleanShot 2024-08-31 at 09 05 09@2x](https://github.com/user-attachments/assets/36d7c998-d1bc-41a5-a036-b1766d1db203)

Does anybody know if the naming convention has changed? Or where we can find any documentation on the current container naming conventions for Frigate?

### Version

dev-2dc5a7f-standard-arm64

### Frigate config file

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
yes, they changed https://github.com/blakeblackshear/frigate/pull/13318

**Comment by victorhooi** [comment]
Got it - thanks for the link to the PR.

I'm still a bit confused - I see there's containers with the prefix `master` - are those the equivalent of the old `dev` containers? (e.g. `master-90221e8-standard-arm64`). Is it just the main branch?

Or should I be looking at the ones that begin with just the hash? (e.g. `6a0b5c3-standard-arm64`)

And what do the images beginning with `cache` mean please? (e.g. ` cache-h8l` - I assume the h8l is the new support for the hailo AI chips? 😀)

**Reply by NickM-27** [reply]
the master- is just from before the change was merged in to master. There will no longer be any branch names included in the builds. You'll need to look at the commit to see which branch it came from
